k8s/jiva/snapshot-cleanup.sh

```diff
@@ -0,0 +1,91 @@
+    #!/bin/bash
+
+    set -x
+
+    usage()
+    {
+        echo "Usage: bash snapshot-cleanup.sh <pv-name> <number_of_snapshots_to_delete>"
+        exit 1
+    }
+
+    warning()
+    {
+        echo "WARNING: This script need to be run from client that have access to the cluster. This script can run for longer time based on the amount of data in the snapshots that need to be deleted. Please don't cancel the operation while it is running. If the script gets broken due to any network issue, you need to run following command [ bash snapshot-cleanup.sh <pv-name> restore_service ]"
+        echo "Do you want to continue (Y/N)"
+        read access
+        if [ $access == "N" ]; then
+        exit 1;
+        fi
+    }
+    delete_jiva_snapshot()
+    {   
+        warning
+        min_required_snapshot=4
+        count=0
+        snapshot_number_cmd='kubectl exec -it $ctrl_pod_name -n $pvc_namespace -- jivactl snapshot ls | grep -v ID | wc -l'
+        snapshot_name_cmd='kubectl exec -it $ctrl_pod_name -n $pvc_namespace -- jivactl snapshot ls | grep -v ID | tail -1'
+
+        if [ $(eval $snapshot_number_cmd) -le $((number_of_snapshot-1)) ]; then
+            echo "Provided number of snapshots are not present Requested: $number_of_snapshot Found: $(eval $snapshot_number_cmd)"
```

Comment:
Error: You have requested to delete `input number`. There are only `current snapshot that can be deleted` snapshots available on this volume. Please re-run this command with by specifying the number of snapshots to be deleted as `current snapshots` or less.